src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Toilet.java

```diff
@@ -1,20 +1,27 @@
-package commodities;
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
+
+/**
+ * Class bed has 2 private members and one inherited from the abstract class AbstractCommodity
+ * methods - getters,setters,constructor and implemented method prepare()
+ */
 
 public class Toilet extends AbstractCommodity {
 
     private boolean hasWaterSavingTechnology;
```

Comment:
what is the idea for adding this field?

Reply:
it's like an 'extra' to the toilet

Reply:
Ok, another interesting one could be to have heated seats for the winter ;)